docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +539,44 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+
+    class Meta:
+        model = Team
+        fields = ("username", "organization")
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.CharField()
+
+    def to_internal_value(self, data):
+        validated_data = super().to_internal_value(data)
+        email_or_username = data.get("member")
+        existing_username = self.context["view"].kwargs.get("member_username")
+
+        if existing_username is not None:
+            if not email_or_username:
+                raise serializers.ValidationError(
+                    {"member": "Username must be provided"}
+                )
+
+            try:
+                if User.objects.fast_search(email_or_username) > 0:
```

Comment:
What is the point of this line? Fast search is going to throw if the email username/email does not exist. You are checking a function that returns type `User` with `int`.

I am pretty sure we already have discussed this.

Reply:
Changed